src/test/scala/org/constellation/domain/checkpointBlock/CheckpointBlockDoubleSpendCheckerTest.scala

```diff
@@ -25,7 +25,7 @@ class CheckpointBlockDoubleSpendCheckerTest extends FunSuite with BeforeAndAfter
   }
 
   test("it should return transaction when transaction's ordinal number is less than or equal the last accepted one") {
-    val firstTransaction = createTransaction("src", "firstTransaction", 1)
+    val firstTransaction = createTransaction("src", "abc123", 1)
```

Comment:
Just a heads up that we can rely on the default value of "abc123" here

Reply:
Do you mean that we can expect it to be base16? If yes I would add simple static validation when submitting a transaction ensuring that prevHash can be parsed as base16.